### ISSUE TITLE
508 Issue: EPA Menu - Tab order is not logical

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#2365]
When tabbing through the ECMPS Screen, in the EPA menu, the focus does not follow a logical order which may confuse users. This issue is a violation of Section 508 and WCAG 2.0 Success Criterion 2.4.3

The Search EPA form field is the last to receive keyboard focus in the menu, but it’s visually the first item:

![image.png](https://images.zenhubusercontent.com/610014e44c513c0bad9f811c/6738533c-d6a5-4e21-ab64-070591160ce5)

**Comment by aprematta** [comment]
Testing complete, both tab and html order is now logical, closing ticket
